packages/components/src/components/header/header.stories.ts

```diff
@@ -7,9 +7,21 @@ const { argTypes, parameters } = storybookDefaults('sd-header');
 const { generateTemplate } = storybookTemplate('sd-header');
 const { overrideArgs } = storybookHelpers('sd-header');
 
+/**
+ * Provides a container for the main navigation of a website.
```

Comment:
Used as a container for the main navigation of a website.

Reply:
Updated